src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -1062,3 +1066,63 @@ def wrapper(*args, **kwargs):
         return result
 
     return wrapper
+
+
+def execute_flow(
+    flow_file: Path,
+    working_dir: Path,
+    output_dir: Path,
+    connections: dict,
+    inputs: Mapping[str, Any],
+    *,
+    func: Optional[str] = None,
+    storage: Optional[AbstractRunStorage] = None,
+    raise_ex: bool = True,
+    stream_output: bool = False,
```

Comment:
Let's only use stream_output parameter

Reply:
If we don't need to run the aggregation nodes, maybe we can use `exec_line` directly. I think this function is used to run the entire flow.

Reply:
Added